detectors/node/opentelemetry-resource-detector-alibaba-cloud/package.json

```diff
@@ -56,7 +56,7 @@
     "@opentelemetry/api": "^1.0.0"
   },
   "dependencies": {
-    "@opentelemetry/resources": "^1.0.0",
+    "@opentelemetry/resources": "^1.10.0",
```

Comment:
note for reviewer: `DetectorSync` interface appeared in `v1.10.0`. this is a leftover from #2328